src/ops/log.rs

```diff
@@ -44,8 +47,23 @@ fn log_other(state: &mut State, _term: &mut Term, _args: &[OsString], result: &s
 fn goto_log_screen(state: &mut State, rev: Option<Oid>) {
     state.screens.drain(1..);
     let size = state.screens.last().unwrap().size;
+    let limit = state
+        .pending_menu
+        .as_ref()
+        .map(|m| m.args.get("-n"))
+        .flatten()
+        .map(Arg::get_i32)
+        .flatten()
```

Comment:
I *think* this results in the same behaviour!
```suggestion
        .and_then(|m| m.args.get("-n"))
        .and_then(Arg::get_i32)
```